scripts/pdf/render.py

```diff
@@ -67,12 +67,16 @@ def main(loc, colorscheme):
 
     html += "</body></html>"
 
+    # Determine the output filename
+    if output_filename is None:
+        output_filename = "tldr-pages.pdf"
```

Comment:
From the other comment, if we apply the default earlier, then don't need to bother with this check.

```suggestion
```